virttest/virsh.py

```diff
@@ -714,6 +714,28 @@ def command(cmd, **dargs):
     return ret
 
 
+def convert_to_dict(result):
+    """
+    Convert virsh command output to a dict.
+
```

Comment:
Can any virsh command output be converted dict?
Or add one reminder that this is specific to virsh command output?

Reply:
Aha, nice catch, I think no. Converting the output of 'virsh dumpxml' to a dict is meaningless I think. I may move it back to libvirt_cgroup later if there is no better choice.

Reply:
BTW, I will give an example in next push.

Reply:
docstring updated.